tests/test_queue.py

```diff
@@ -55,7 +55,7 @@ def test_handle_get_stats(
 def test_important_files_are_analyzed_first(create_task_queue, mocker, repo):
     enqueue = mocker.patch("seagoat.queue.task_queue.TaskQueue.enqueue")
     create_task_queue()
-    sleep(0.5)
+    sleep(2.5)
```

Comment:
Some other tests were failing because of this timeout, at least on my laptop. So I increased it.

Reply:
sure! but does it need to be increase all the way to 2.5s? if so, I'm ok with this change of course, it's important for the tests to run properly on any reasonable computer

Reply:
Managed to get it down 2.0, but not lower.